packages/one-app-runner/__tests__/src/startApp.spec.js

```diff
@@ -41,33 +41,29 @@ describe('startApp', () => {
     delete process.env.HTTP_ONE_APP_DEV_PROXY_SERVER_PORT;
     delete process.env.HTTP_METRICS_PORT;
     delete process.env.NODE_EXTRA_CA_CERTS;
-    jest.spyOn(process.stdout, 'write');
-    jest.spyOn(process.stderr, 'write');
-    jest.spyOn(require('fs'), 'createWriteStream');
   });
 
-  afterAll(() => {
-    fs.unlinkSync(pathToLogFile);
-  });
-
-  it('pulls one app docker image and starts one app', () => {
-    const mockSpawn = require('mock-spawn')();
+  it('pulls one app docker image and starts one app', async () => {
+    expect.assertions(1);
+    const mockSpawn = makeMockSpawn();
     childProcess.spawn.mockImplementationOnce(mockSpawn);
-    startApp({ moduleMapUrl: 'https://example.com/module-map.json', rootModuleName: 'frank-lloyd-root', appDockerImage: 'one-app:5.0.0' });
+    await startApp({ moduleMapUrl: 'https://example.com/module-map.json', rootModuleName: 'frank-lloyd-root', appDockerImage: 'one-app:5.0.0' });
```

Comment:
These changes were required to resolve run away promises that threw error in node 16